ToDoLite/DetailViewController.m

```diff
@@ -226,6 +246,18 @@ - (UITableViewCell *)couchTableSource:(CBLUITableSource *)source cellForRowAtInd
                                                     forIndexPath:indexPath];
     CBLQueryRow *row = [source rowAtIndex:indexPath.row];
     Task *task = [Task modelForDocument:row.document];
+
+    int64_t delay = 1.5;
+    // avoid having all text start gray only to immediately turn black (otherwise there is a flicker effect that users might think is a bug)
+    dispatch_time_t time = dispatch_time(DISPATCH_TIME_NOW, delay * NSEC_PER_SEC);
+    dispatch_after(time, dispatch_get_main_queue(), ^(void){
+        if ([repl isDocumentPending:task.document]) {
+            cell.name.textColor = [UIColor grayColor];
+        } else {
+            cell.name.textColor = [UIColor blackColor];
+        }
+    });
+
```

Comment:
Oops, you're right, I'd only tested with a few items. [I got rid of the delay](https://github.com/PaulCapestany/ToDoLite-iOS/commit/ff3b48ada34e312e83c63eca43557c8f15556fc6), but now there's a flicker, so yeah, not sure what the best way to deal with that might be yet..